ingress/controllers/nginx/nginx.tmpl

```diff
@@ -92,6 +92,13 @@ http {
     resolver {{ .defResolver }} valid=30s;
     {{ end }}
 
+    {{/* Whenever nginx proxies a request without a "Connection" header, the "Connection" header is set to "close" */}}
+    {{/* when making the target request.  This means that you cannot simply use */}}
+    {{/* "proxy_set_header Connection $http_connection" for WebSocket support because in this case, the */}}
+    {{/* "Connection" header would be set to "" whenever the original request did not have a "Connection" header, */}}
+    {{/* which would mean no "Connection" header would be in the target request.  Since this would deviate from */}}
+    {{/* normal nginx behavior we have to use this approach. */}}
```

Comment:
@bprashanth I think this a better place because the next map changes the http behavior and the comment show the why and consequences of changing the values